README.md

```diff
@@ -1301,6 +1301,7 @@ CyberTalks</b></a> - talks, interviews, and article about cybersecurity.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://github.com/gentilkiwi/mimikatz"><b>mimikatz</b></a> - a little tool to play with Windows security.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://github.com/sherlock-project/sherlock"><b>sherlock</b></a> - hunt down social media accounts by username across social networks.<br>
```

Comment:
Hi

Reply:
Hello, can someone merge this PR, someone having write access can give an approving review please? Thanks.

Reply:
Good and Excellent work

Reply:
Thank you @TalleA15. Actually, the message says, "At least 1 approving review is required by reviewers with write access."
So, if anyone of you guys @Nader822 @Mendoza83 @TalleA15 @Theforgottenisme is having write access please approve the change and it will get merged. Thank you.